identifiers/country-gb/regions-eng.csv

```diff
@@ -0,0 +1,9 @@
+ocd-division/country:gb/country:gb-eng/region:ukc,North East
+ocd-division/country:gb/country:gb-eng/region:ukd,North West
+ocd-division/country:gb/country:gb-eng/region:uke,Yorkshire and the Humber
+ocd-division/country:gb/country:gb-eng/region:ukf,East Midlands
+ocd-division/country:gb/country:gb-eng/region:ukg,West Midlands
+ocd-division/country:gb/country:gb-eng/region:ukh,Eastern
+ocd-division/country:gb/country:gb-eng/region:uki,Greater London
+ocd-division/country:gb/country:gb-eng/region:ukj,South East
+ocd-division/country:gb/country:gb-eng/region:ukk,South West
```

Comment:
Are these common abbreviations for the regions? (ukc, ukd,...)

Reply:
Yes, these are the NUTS-Codes for the Regions.
https://en.wikipedia.org/wiki/NUTS_1_statistical_regions_of_England
http://statoids.com/ugb.html

Reply:
@jpmckinney any opinion here? Because you wrote above: "NUTS is for statistical geography, whereas the project is for political geography".
I didn't find any other shortcodes to describe the regions. Or should we use the long names from there regions instead of the NUTS Codes?

ocd-division/country:gb/part:eng/region:west_midlands,West Midlands ?

Reply:
I think it's fine to re-use codes from statistical geography if they align with political geography. In this case, the statistical regions seem to have been defined based on political regions, so it's fine to use NUTS. I don't see any suitable alternatives in e.g. ISO 3166-2.